domains/manhha.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "haconbk"
+    },
+    "record": {
+        "CNAME": "haconbk.github.io"
```

Comment:
Little content provided on site. Reopen this PR when you have completed your site.